lib/rouster.rb

```diff
@@ -188,7 +188,29 @@ def initialize(opts = nil)
 
         raise ArgumentError.new('AWS passthrough requires valid :sshkey specification, should be path to private half') unless File.file?(@passthrough[:key])
         @sshkey = @passthrough[:key]
+      elsif @passthrough[:type].eql?(:openstack)
+        @logger.debug(sprintf('instantiating an %s passthrough worker', @passthrough[:type]))
+        @sshkey = @passthrough[:key]
+
+        ostack_defaults = {
+          :ssh_port              => 22,
+          :user                  => 'vagrant',
```

Comment:
is this the right default?

in searching around, it seems that like AWS, the 'expected' username varies with the image itself, and it seems to be OS specific.

my preference would be to remove the default here entirely